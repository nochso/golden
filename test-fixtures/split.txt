this part will not have a new line
===
this one will have a trailing one

===

this one has a leading one
===

this one has both

===


two new lines each


===this is less readable but works===this too

===end