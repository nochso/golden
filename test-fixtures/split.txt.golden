[]string{
  "this part will not have a new line",
  "this one will have a trailing one\n",
  "\nthis one has a leading one",
  "\nthis one has both\n",
  "\n\ntwo new lines each\n\n",
  "this is less readable but works",
  "this too\n",
  "end",
}